hrm.py

```diff
@@ -171,25 +186,38 @@ def anomaly(time, brady_thresh, brady_time, tachy_thresh, tachy_time):
     tachyTimes = []
     counter = 0
     for i in range(1, len(time)):
-        if 60000 / (time[i] - time[i-1]) < brady_thresh and dying_slow == 0:
+        if (60000 / (time[i] - time[i - 1]) < brady_thresh) and \
```

Comment:
60000 should likely be defined as a constant at the top of the file in a descriptive way. an example could be:
```python
SECONDS_IN_MIN = 60
```

Reply:
Should we still fix hrm.py if we're not using it in class_based approach?

Reply:
Also for the class-based, would this be a global variable outside of the class?

Reply:
@cshackerpro focus on getting the class based approach fixed first and then circle back to `hrm.py` to clean things up if you have time (it shouldn't take too long). 

Yep, this would be a global variable outside the class. Among many reasons for doing this, this ensures this constant isn't reallocated in memory for every new instantiation of your class. It also ensures all class instances are using the same constant (less risk of one class changing it's instance variable constant and having others use them, although ideally this would be a static constant but python isn't really into that :( )